plugins/chat/app/serializers/chat/thread_original_message_serializer.rb

```diff
@@ -1,43 +1,26 @@
 # frozen_string_literal: true
 
 module Chat
-  class ThreadOriginalMessageSerializer < Chat::MessageSerializer
-    has_one :user, serializer: BasicUserWithStatusSerializer, embed: :objects
-
-    def excerpt
```

Comment:
Not sure why we dropped the use of `censored_excerpt` here, it's running it through WatchedWord

Reply:
I added it back